### PR TITLE
make mail_exclude_filter test more verbose

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by Lex-2008** [comment]
@cf-bottom can you request a Jenkins build for me here, please?

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4014/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4014/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4014/

File: tests/acceptance/25_cf-execd/mailfilter_common.cf.sub [hunk 1]
```diff
@@ -142,10 +142,10 @@ bundle agent compare_cf_execd_mail(test)
...
 
   reports:
-#      "unwanted_regex_list = $(unwanted_regex_list)";
-#      "wanted_regex = '$(wanted_regex)'";
-#      "unwanted_regex = '$(unwanted_regex)'";
-#      "output = '$(output)'";
+      "unwanted_regex_list = $(unwanted_regex_list)";
+      "wanted_regex = '$(wanted_regex)'";
+      "unwanted_regex = '$(unwanted_regex)'";
+      "output = '$(output)'";
```
Comment (olehermanse) [inline tests/acceptance/25_cf-execd/mailfilter_common.cf.sub]: Will these be printed when the test fails in jenkins?

Comment (olehermanse) [reply tests/acceptance/25_cf-execd/mailfilter_common.cf.sub]: @Lex-2008 please answer

Comment (Lex-2008) [reply tests/acceptance/25_cf-execd/mailfilter_common.cf.sub]: That's my expectation, yes.

**Pull request merged**